assets/home/tasks-section/first-course.test.js

```diff
@@ -29,8 +29,8 @@ describe( '<FirstCourse />', () => {
 
 		expect( queryByText( 'SITE_TITLE' ) ).toBeTruthy();
 		expect( queryByText( 'COURSE_TITLE' ) ).toBeTruthy();
-		expect( siteLogo ).toEqual( 'SITE_LOGO' );
-		expect( featuredImage ).toEqual( `url(FEATURED_IMAGE)` );
+		expect( siteLogo ).toEqual( 'SITE_IMAGE' );
+		expect( featuredImage ).toEqual( `url(COURSE_IMAGE)` );
```

Comment:
Small suggestion, but WDYT about changing the `featuredImage` const to `courseImage`?

Reply:
Good catch! https://github.com/Automattic/sensei/pull/5946/commits/aeb8911298a7fce8ce48764fb0a0fb966162ce44